extendr-api/src/robj/into_robj.rs

```diff
@@ -763,20 +819,6 @@ impl From<Vec<Rstr>> for Robj {
     }
 }
 
-impl From<Vec<Rint>> for Robj {
-    /// Convert a vector of Rint into integers.
-    fn from(val: Vec<Rint>) -> Self {
-        Integers::from_values(val.into_iter()).into()
-    }
-}
-
-impl From<Vec<Rfloat>> for Robj {
-    /// Convert a vector of Rfloat into doubles.
-    fn from(val: Vec<Rfloat>) -> Self {
-        Doubles::from_values(val.into_iter()).into()
-    }
-}
-
```

Comment:
Is there a corresponding test to ensure that one can still create `Integers` and `Doubles` from a `Vec<Rint>` or `Vec<Rfloat>`?